src/components/epoch-countdown/epoch-countdown.tsx

```diff
@@ -7,7 +7,7 @@ import { useTranslation } from "react-i18next";
 
 import arrow from "../../images/back.png";
 
-export const DATE_FORMAT = "yyyy.MM.dd HH:mm";
+const DATE_FORMAT = "dd MMMM yyyy HH:mm";
```

Comment:
Can we exportn this as a const from some file rather than redefining it everywhere. I think in console we have something like:
* `DATE_FORMAT`
* `SHORT_DATE_FORMAT`